core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -103,13 +105,22 @@ public void setScm(Scm scm) {
         this.scm = scm;
     }
 
+    public Path getCustomStrategiesDefinition() {
+        return customStrategiesDefinition != null ? Paths.get(customStrategiesDefinition) : null;
```

Comment:
This is probably not the best way of getting the path - don't forget that in case it is called from other than the main directory, you could get a wrong path. eg:

`mvn clean package -f ./smart-testing-test/pom.xml -Dsmart.testing.custom.strategies=./my-project/strategy.properties`

Reply:
But then we start the discussion of is `.` referring to project root or `pom.xml` root?

Reply:
Project root is the same as `pom.xml` root, isn't it? 
You can call the project from a different directory using the parameter `-f` (eg. from your home directory)

Reply:
Ok then you mean it is better to use `baseDir` as well?

Reply:
`baseDir` wouldn't help you.
Anyway, I would keep it only embedded https://github.com/arquillian/smart-testing/pull/212#discussion_r143196654 so you don't have to care about this path thingy

Reply:
True 😆